### DISCUSSION TITLE
inflight node module, should we be worried or ignore?

### DISCUSSION THREAD

**Discussion by bbbjames** [FrancescoXX/rustcrab#28] [Q&A]
![now](https://github.com/user-attachments/assets/9d94c9c2-6af0-4faa-8674-accc9355638c)

    "node_modules/inflight": {
      "version": "1.0.6",
      "resolved": "https://registry.npmjs.org/inflight/-/inflight-1.0.6.tgz",
      "integrity": "sha512-k92I/b08q4wvFscXCLvqfsHCrjrF7yiXsQuIVvVE7N82W3+aqpzuUdBbfhWcy/FZR3/4IgflMgKLOsvPDrGCJA==",
      "deprecated": "This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.",
      "dev": true,
      "dependencies": {
        "once": "^1.3.0",
        "wrappy": "1"
      }

**Comment by FrancescoXX** [comment]
thank you @bbbjames . can you open an issue on this?

**Comment by FrancescoXX** [comment]
@bbbjames is this solved?
